docs/l10nScreenshotsTests.md

```diff
@@ -2,14 +2,30 @@
 We are using [`screengrab`](https://docs.fastlane.tools/getting-started/android/screenshots/) which works with fastlane to automate the process of capturing screenshots. 
 All the l10n screenshots are generated through the ui tests. These particular tests run as part of the screenshots package (`app/src/androidTest/mozilla/fenix/ui/screenshots`)
 
-### Steps to Run / Build
+### Run tests locally from Android Studio
+Navigate to `app/src/androidTest/mozilla/fenix/ui/screenshots`, once in that directory, run the full test suite or a specific test by clicking on the `>` button.
+
+By running them manually you can check whether the test works or not but screenshots will not be saved. 
+
+### Run tests locally from command line
 1. Install the gem:
 `sudo gem install screengrab`
 
 2. From command line run: 
 `fastlane screengrab --test_instrumentation_runner "androidx.test.runner.AndroidJUnitRunner"` 
 
-Screenshots will be saved in the root directory: `fastlane/metadata/android` 
+The package configuration, apk paths as well as the locales are set in [Screengrab file](https://github.com/mozilla-mobile/fenix/blob/073fd8939067bc7a367d8db497bcf53fbd24cdd2/fastlane/Screengrabfile#L5).
+In case there is a change there the file has to be modified accordingly.
+Before launching that command, there has to be an emulator running.
+
+Once the build and tests finsish, screenshots will be saved in the root directory: `fastlane/metadata/android`
 If there is a failure and screenshots are not saved, it may be necessary to create these folders manually first.
 Currently screenshots are uploaded to [`gcloud`](https://storage.googleapis.com/l10n-fenix/2019-06-21/screenshots.html) once they are generated manually by running the previous command line.
```

Comment:
Is this still the case? or would these be retained in taskcluster?

Reply:
oh I should remove that, right, thanks